models.py

```diff
@@ -12,3 +16,35 @@ def mostrecent(x):
     preds = X.apply(mostrecent, axis=1)
 
     return preds
+
+def arima(X):
+    """Predict the most recent value using ARIMA(2,1,0) model.
+
+    If any issues fitting time series model, use status_quo."""
+    
+    sq_preds = status_quo_model(X)
+    X["id"] = X.index
+    # get it to long form
```

Comment:
Do you think the preprocessing here would be useful for other methods? If so, best to add it to the dataset class so that we can reuse it.